bedrock/mozorg/templates/mozorg/contribute/signup-new.html

```diff
@@ -0,0 +1,99 @@
+
+{% block contrib_title %}
+  <div class="container">
+    <h1 class="page-title">{{ _('Get Involved with Mozilla Today') }}</h1>
+  </div>
+{% endblock %}
+
+
+{% block contrib_content %}
+<article class="container">
+
+  <section>
+    <header>
+      <h2>{{ _('Mozilla Participation Tasks') }}</h2>
+    </header>
+
+    <ul class="simple">
+      <li class="task">
+        <h3>{{ _('Customize Your Desktop Browser') }}</h3>
+        <a href="{{ url('mozorg.contribute.tasks.whimsey') }}" class="do-task more" role="button">{{ _('Add to Firefox') }}</a>
+        <div class="tooltip hidden">
+          <p>{{ _('Whimsy is a fun Firefox desktop add-on and experiment with subtle, whimsical things happening in the browser.') }}</p>
+        </div>
+      </li>
+
+      <li class="task">
+        <h3 class="fx-mobile">{{ _('Use Firefox on Your Phone') }}</h3>
+        <a href="{{ url('mozorg.contribute.tasks.firefox-mobile') }}" class="do-task more" role="button">{{ _('Download Firefox') }}</a>
+        <div class="tooltip hidden">
+          <p>{{ _('Firefox is the independent, people-first browser with the privacy and features you love, now available on Android and iOS!') }}</p>
+        </div>
+      </li>
+
+      <li class="task">
+        <h3 class="fx-twitter">{{ _('Connect on Twitter') }}</h3>
+        <a href="{{ url('mozorg.contribute.tasks.follow-mozilla') }}" class="do-task more" role="button">{{ _('Follow Mozilla') }}</a>
+        <div class="tooltip hidden">
+          <p>{{ _('Stay connected with us for news, announcements and opportunities from the Firefox team feed.') }}</p>
+        </div>
+      </li>
+    </ul>
+  </section>
+
+  <section>
+    <header>
+      <h2>{{ _('Mozilla Bigger Participation Tasks') }}</h2>
+    </header>
+    <ul class="challenging">
+      <li class="task">
+        <h3 class="coding">{{ _('Watch Someone Live Hack on Firefox') }}</h3>
+        <a href="{{ url('mozorg.contribute.tasks.joy-of-coding') }}" class="do-task more" role="button">{{ _('Watch the video') }}</a>
+        <div class="tooltip hidden">
+          <p>{{ _('Firefox engineer mconley livehacks and crushes bugs where he finds them in this unscripted, uncensored, true-to-life video series.') }}</p>
+        </div>
+      </li>
+
+      <li class="task">
+        <h3 class="fxos">{{ _('Help Build Firefox OS') }}</h3>
+        <a href="{{ url('mozorg.contribute.tasks.build-firefox') }}" class="do-task more" role="button">{{ _('Start Foxfooding') }}</a>
+        <div class="tooltip hidden">
+          <p>{{ _('If you’re interested in new skills (or adept at porting), get involved in this ground-up project to eat our own fox food by building mobile Firefox OS together!') }}</p>
+        </div>
+      </li>
+
+      <li class="task">
+        <h3 class="devtools">{{ _('Learn a Bit About Coding') }}</h3>
+        <a href="{{ url('mozorg.contribute.tasks.devtools-challenger') }}" class="do-task more" role="button">{{ _('Take the challenge') }}</a>
```

Comment:
Since these `<a>` tags actually navigate to a different page (except the Whimsy installer, which will be an iframed button), should they have the `role="button"` attribute?